### DISCUSSION TITLE
阿呆读新闻

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1541] [General]
这是阿呆的新闻栏目，主楼用于发布新闻链接，路人请勿在主楼插队（可在每层楼内评论）。

**Comment by yihui** [comment]
[【一周要闻】 鸦鹊岭亮点工作一周资讯 (11月13日-11月19日)](https://mp.weixin.qq.com/s/9OL5Gz4SSOFcHBzDlSUhtg)

**Reply by zhijix** [reply]
我觉得救火演习好好玩！

**Comment by yihui** [comment]
[致全镇广大居民朋友冬季安全的一封信](https://mp.weixin.qq.com/s/XmEm0U-z1tJ70gfkBrfNWg)

**Reply by zhijix** [reply]
我没有感想………………………………………………………………………………………………………………………………………………。

**Comment by yihui** [comment]
[爷爷家的桔子卖不出去了，咋办呢？](https://mp.weixin.qq.com/s/JbisVYb5zgPRio6cWyooPA)

**Reply by zhijix** [reply]
Write a reply. Write a reply. Write a reply. Write a reply. Write a reply. Write a reply. Write a reply. Write a reply. Write a reply. Write a reply. Write a reply. Ok, a reply.

**Reply by yihui** [reply]
同样的话写十一遍你不觉得无聊吗？

爷爷的桔子卖不出去，你能不能想个好办法帮他的忙？

**Comment by zhijix** [comment]
你得四点半做饭要不然我就不会对你这么好了😁

**Reply by yihui** [reply]
哼，说的就像对我好过一样。以后要多帮我削土豆皮、洗菜、切菜、煮米饭、炒花生米。这样我可以考虑四点半开始做饭。

今天回家不要忘记了打五十个字。你昨天看见有五个读音的字是“欸”还是“诶”？

另外，给你看个新闻（[点这里](https://mp.weixin.qq.com/s/NwS_vpReRrjqPVfOR2oqDg)），是我以前的高中，有个英国人跑过去给学生们做报告，还当场做了化学实验，烧起了火，不知道是怎么弄的。看着好像有点好玩的样子。
